addons/base_import/static/src/xml/base_import.xml

```diff
@@ -97,6 +100,12 @@
         </tbody>
     </t>
 
+    <t t-name="ImportView.previewPopover">
```

Comment:
Strange, other templates are not using the same typing ... Maybe "ImpactView.preview_popover" ? That way it matches file style.

Would also move if after  "comment_*" templates (to order them a bit).

Reply:
(I'm guessing Thibault meant "ImportView.preview_popover")